### PR TITLE
New landing page for Streams API

### PR DESCRIPTION
**Pull request by derrickdoo**
New Kafka Streams API landing page for desktop and mobile web.

image

Note: There are a couple of items that need to be filled in before we launch this page.

**Fill in links. Placeholders added for:**
1. Write your first app
2. View Demo
3. Tutorials

**Add desired Scala and Java 7 code snippets**

### CODE REVIEW

File: 0110/streams/index.html [hunk 1]
```diff
@@ -18,56 +18,192 @@
...
+                        config.put(StreamsConfig.APPLICATION_ID_CONFIG, "wordcount-application");
+                        config.put(StreamsConfig.BOOTSTRAP_SERVERS_CONFIG, "kafka-broker1:9092");
+                        config.put(StreamsConfig.DEFAULT_KEY_SERDE_CLASS_CONFIG, Serdes.String().getClass());
+                        config.put(StreamsConfig.DEFAULT_VALUE_SERDE_CLASS_CONFIG, Serdes.String().getClass());
+
+                        KStreamBuilder builder = new KStreamBuilder();
+                        KStream&lt;String, String&gt; textLines = builder.stream("TextLinesTopic");
+                        KTable&lt;String, Long&gt; wordCounts = textLines
+                            .flatMapValues(new ValueMapper&lt;String, Iterable&lt;String&gt;&gt;() {
+                            @Override
```
Comment (miguno) [inline 0110/streams/index.html]: The indentation of this Java 7 code example isn't quite right.  Same for the Scala code example.

See https://gist.github.com/miguno/786b1585c344791c325df325de190355 how it should look like.

File: 0110/streams/index.html [hunk 2]
```diff
@@ -18,56 +18,192 @@
...
+                import org.apache.kafka.common.serialization._
+                import org.apache.kafka.streams._
+                import org.apache.kafka.streams.kstream.{KStream, KStreamBuilder, KTable}
+
+                import scala.collection.JavaConverters.asJavaIterableConverter
+
+                object WordCountApplication {
+
+                    def main(args: Array[String]) {
+                        val config: Properties = {
```
Comment (miguno) [inline 0110/streams/index.html:175]: The indentation of this Scala code example isn't quite right.

See https://gist.github.com/miguno/e3c791f67068efa0cd741d8929a9662a how it should look like.

Comment (derrickdoo) [reply 0110/streams/index.html:175]: Thanks, good catch. Should be fixed now.

**Comment by guozhangwang** [comment]
LGTM. Merging to asf-site.

@derrickdoo Could you file a corresponding PR against `kafka` trunk branch? I will merge it then to both trunk and 0.11.0 branches.

**Pull request closed without merging**